pkg/providers/pricing/pricing.go

```diff
@@ -16,8 +16,22 @@ package pricing
 
 import (
 	"context"
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
 	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strconv"
 	"sync"
+
+	"k8s.io/klog/v2"
+)
+
+const (
+	initialOnDemandPricesFile = "initial-on-demand-prices.json"
+	pricingCSVURL             = "https://gcloud-compute.com/machine-types-regions.csv"
```

Comment:
Add one todo, we will implement it in a better way

Reply:
@jwcesign Added, but I'm not sure if I wrote what you meant. So please check